File: ApiSuvesaPos/Negocio/Modelo/cxc/apartados/apartados.vb
Namespace Modelo
    Public Class apartados
        Public Property Id_Apartado As Long
        Public Property Id_Cliente As Long
        Public Property Nombrecliente As String
        Public Property Cedulausuario As String
        Public Property SubTotal As Decimal
        Public Property Imp_Venta As Decimal
        Public Property Total As Decimal
        Public Property Fecha As DateTime
        Public Property Vence As DateTime
        Public Property Anulado As Boolean
        Public Property Cancelado As Boolean
        Public Property SubTotalGravada As Decimal
        Public Property SubTotalExento As Decimal
        Public Property Observaciones As String
        Public Property Monto As Decimal
        Public Property Descuento As Decimal
        Public Property Tipo_Cambio As Decimal
        Public Property Cod_Moneda As Integer
        Public Property Apartado_Detalle As New List(Of Modelo.apartado_detalle)
    End Class

End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/apartados/apartados.cs
namespace Modelo
{
    public class apartados
    {
        public long Id_Apartado { get; set; }
        public long Id_Cliente { get; set; }
        public string Nombrecliente { get; set; }
        public string Cedulausuario { get; set; }
        public decimal SubTotal { get; set; }
        public decimal Imp_Venta { get; set; }
        public decimal Total { get; set; }
        //public DateTime Fecha { get; set; }
        //public DateTime Vence { get; set; }
        public bool Anulado { get; set; }
        public bool Cancelado { get; set; }
        public decimal SubTotalGravada { get; set; }
        public decimal SubTotalExento { get; set; }
        public string Observaciones { get; set; }
        public decimal Monto { get; set; }
        public decimal Descuento { get; set; }
        public decimal Tipo_Cambio { get; set; }
        public int Cod_Moneda { get; set; }
        //public List<Modelo.apartado_detalle> Apartado_Detalle { get; set; } = new List<Modelo.apartado_detalle>();
    }
}
